app/assets/javascripts/components/StickyColumn.js

```diff
@@ -0,0 +1,215 @@
+define(['jquery', 'DoughBaseComponent', 'eventsWithPromises'], function($, DoughBaseComponent, eventsWithPromises) {
+  'use strict';
+
+  var StickyColumn,
+      defaultConfig = {},
+      $parent,
+      contentHeight,
+      topMargin,
+      top,
+      bottom,
+      isFixed,
+      isAtBottom,
+      isInSidebar,
+      mainContentSelector = '.l-article-3col-main',
+      parentSelector = '.l-article-3col-right',
+      fixedClass = 'related-links--desktop-fixed',
+      bottomClass = 'related-links--desktop-bottom';
+
+  StickyColumn = function($el, config) {
+    StickyColumn.baseConstructor.call(this, $el, config, defaultConfig);
+  };
+
+  /**
+  * Inherit from base module, for shared methods and interface
+  */
+  DoughBaseComponent.extend(StickyColumn);
+
+  /**
+   * [init description]
+   * @param  {[type]}
+   * @return {[type]}
+   */
+  StickyColumn.prototype.init = function(initialised) {
+    this._initialisedSuccess(initialised);
+
+    this._bindEvents();
+    this._measureDom();
+    this._showElement();
+    this._positionComponent();
+
+    return this;
+  };
+
+  /**
+   * [_bindEvents description]
+   * @return {[type]}
+   */
+  StickyColumn.prototype._bindEvents = function() {
+    $(window)
+    .on('resize', this._debounce($.proxy(this._handleResize, this), 200))
+    .on('scroll', $.proxy(this._handleScroll, this));
+
+    eventsWithPromises.subscribe('toggler:toggled', $.proxy(this._handleSectionToggle, this));
+  };
+
+  /**
+   * [_isMQMedium description]
+   * @return {Boolean}
+   */
+  StickyColumn.prototype._isMQMedium = function() {
```

Comment:
`_isMQMedium` doesn't really describe what this method does. Maybe something like `_isElementVisible(this.$el)` would work better?